src/styles/blocs/header/header_nav.scss

```diff
@@ -0,0 +1,54 @@
+.nav {
+  &__list {
+    display: flex;
+    align-items: center;
+    list-style: none;
```

Comment:
you've added `list-style: none; ` already to reset.scss

you can remove here

Reply:
Ok